ruby/vampires.rb

```diff
@@ -0,0 +1,98 @@
+#Driver code
+#Process multiple employees
+puts "How many employees will be processed?"
+ee_num = gets.chomp.to_i
+#initiating loop to process multi employees. It wraps code
+count = 0
+while count < ee_num
+
+# Program to ask new employee:
+#Some known vampires are in the area, and we can check names against the Werewolf Intelligence Bureau database for their aliases.
+
+puts "What is your name?"
+name = gets.chomp.capitalize #entered info is capitalized as names usually are
+puts "Welcome, " + name + "." #print to screen to confirm info entering program
```

Comment:
String interpolating in Ruby is convention over adding strings.

puts "Welcome, #{name}."